exporting/aws_kinesis/aws_kinesis.c

```diff
@@ -2,6 +2,44 @@
 
 #include "aws_kinesis.h"
 
+/**
+ * Clean AWS Kinesis *
+ */
+void aws_kinesis_cleanup(struct instance *instance)
+{
+    info("EXPORTING: cleaning up instance (aws_kinesis_cleanup) %s ...", instance->config.name);
+    if (instance->buffer)
+        buffer_free(instance->buffer);
+
+    if (instance->engine->aws_sdk_initialized)
+        aws_sdk_shutdown();
```

Comment:
It should be called after `kinesis_shutdown`. Even better to call it in `exporting_clean_engine` because there might be several kinesis instances.

Reply:
Good catch, thank you for the suggestion.